tests/puppeteer/pages/BO/design/pages/addPageCategory.js

```diff
@@ -0,0 +1,41 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class AddPageCategory extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitleCreate = 'Pages';
+
+    // Selectors
+    this.nameInput = '#cms_page_category_name_1';
+    this.displayed = 'label[for=\'cms_page_category_is_displayed_%ID\']';
+    this.descriptionIframe = '#cms_page_category_description_1';
+    this.metaTitleInput = '#cms_page_category_meta_title_1';
+    this.metaDescriptionInput = '#cms_page_category_meta_description_1';
+    this.metaKeywordsInput = '#cms_page_category_meta_keywords_1-tokenfield';
+    this.saveCategoryButton = 'div.card-footer button';
+  }
+
+  /*
+  Methods
+   */
+
+  /**
+   * Fill form for add/edit page category
+   * @param pageCategoryData
+   * @return {Promise<textContent>}
+   */
+  async createEditPageCategory(pageCategoryData) {
+    await this.setValue(this.nameInput, pageCategoryData.name);
+    if (pageCategoryData.displayed) await this.page.click(this.displayed.replace('%ID', '1'));
```

Comment:
Ids should be decalred as constant. why '1', and why '0'? name them for better understanding

Reply:
Is it the ID or the row number ?

Reply:
It is the ID to replace in the selector, I added a comment for explication